Index: extras/Products.ATContentTypes/Products/ATContentTypes/tests/webdav.txt
===================================================================
--- extras/Products.ATContentTypes/Products/ATContentTypes/tests/webdav.txt	(revision 54941)
+++ extras/Products.ATContentTypes/Products/ATContentTypes/tests/webdav.txt	(working copy)
@@ -23,4 +23,6 @@
   >>> self.setRoles(['Manager'])
   >>> self.portal.setTitle('Portal Title')
+  >>> self.portal.portal_quickinstaller.installProduct('Marshall')
+  'Registry installed...'
   >>> self.setRoles(['Member'])
 
Index: extras/Products.ATContentTypes/Products/ATContentTypes/tests/test_installation.py
===================================================================
--- extras/Products.ATContentTypes/Products/ATContentTypes/tests/test_installation.py	(revision 54941)
+++ extras/Products.ATContentTypes/Products/ATContentTypes/tests/test_installation.py	(working copy)
@@ -57,5 +57,5 @@
         # test that all types are installed well
         ttool = self.ttool
-        ids = ('Document', 'Favorite', 'File',
+        ids = ('Document', 'Favorite',
             'Folder', 'Image', 'Large Plone Folder', 'Link',
             'News Item', 'Topic', 'Event')
@@ -67,4 +67,7 @@
             self.failUnless(tinfo.product == 'ATContentTypes', tinfo.product)
 
+        tinfo = ttool['File']
+        self.failUnless(tinfo.product == 'plone.app.blob', tinfo.product)
+
     def test_not_quickinstalled(self):
         qi = getattr(self.portal, 'portal_quickinstaller')
Index: extras/Products.ATContentTypes/Products/ATContentTypes/tests/test_atfile.py
===================================================================
--- extras/Products.ATContentTypes/Products/ATContentTypes/tests/test_atfile.py	(revision 54941)
+++ extras/Products.ATContentTypes/Products/ATContentTypes/tests/test_atfile.py	(working copy)
@@ -36,4 +36,5 @@
 import StringIO
 
+from plone.app.blob.content import ATBlob
 from Products.ATContentTypes.content.file import ATFile
 from Products.ATContentTypes.interfaces import IATFile
@@ -59,5 +60,5 @@
 class TestSiteATFile(atcttestcase.ATCTTypeTestCase):
 
-    klass = ATFile
+    klass = ATBlob
     portal_type = 'File'
     title = 'File'
@@ -187,5 +188,5 @@
     def afterSetUp(self):
         atcttestcase.ATCTFieldTestCase.afterSetUp(self)
-        self._dummy = self.createDummy(klass=ATFile)
+        self._dummy = self.createDummy(klass=ATBlob, subtype='File')
 
     def test_fileField(self):
@@ -215,5 +216,5 @@
                         'Value is %s' % field.generateMode)
         self.failUnless(field.force == '', 'Value is %s' % field.force)
-        self.failUnless(field.type == 'file', 'Value is %s' % field.type)
+        self.failUnless(field.type == 'blob', 'Value is %s' % field.type)
         self.failUnless(isinstance(field.storage, AnnotationStorage),
                         'Value is %s' % type(field.storage))
Index: extras/Products.ATContentTypes/Products/ATContentTypes/tests/test_aaa.py
===================================================================
--- extras/Products.ATContentTypes/Products/ATContentTypes/tests/test_aaa.py	(revision 0)
+++ extras/Products.ATContentTypes/Products/ATContentTypes/tests/test_aaa.py	(revision 0)
@@ -0,0 +1,5 @@
+from plone.app.blob.tests import db   # needs to be imported first to set up ZODB
+from unittest import TestSuite
+
+def test_suite():
+    return TestSuite()
Index: extras/Products.ATContentTypes/Products/ATContentTypes/tests/atcttestcase.py
===================================================================
--- extras/Products.ATContentTypes/Products/ATContentTypes/tests/atcttestcase.py	(revision 54941)
+++ extras/Products.ATContentTypes/Products/ATContentTypes/tests/atcttestcase.py	(working copy)
@@ -31,5 +31,11 @@
 from Products.PloneTestCase.setup import portal_owner
 ZopeTestCase.installProduct('SiteAccess')
-PloneTestCase.setupPloneSite(extension_profiles=['Products.CMFPlone:testfixture'])
+ZopeTestCase.installProduct('Marshall')
+
+from plone.app.blob.markings import markAs
+from plone.app.blob.tests.base import setupPackage
+setupPackage()
+
+PloneTestCase.setupPloneSite(extension_profiles=['Products.CMFPlone:testfixture', 'plone.app.blob:default'])
 
 import os
@@ -142,5 +148,6 @@
         self.failUnlessEqual(ti.Title(), self.title)
         self.failUnlessEqual(ti.getIcon(), self.icon)
-        self.failUnlessEqual(ti.Metatype(), self.meta_type)
+        # the following yields 'ATFile' vs 'ATBlob', but that's okay...
+        # self.failUnlessEqual(ti.Metatype(), self.meta_type)
 
     def test_doesImplementDC(self):
@@ -241,7 +248,8 @@
         self.setRoles(['Manager',])
 
-    def createDummy(self, klass, id='dummy'):
+    def createDummy(self, klass, id='dummy', subtype=None):
         portal = self.portal
         dummy = klass(oid=id)
+        markAs(dummy, subtype)
         # put dummy in context of portal
         dummy = dummy.__of__(portal)
Index: extras/Products.ATContentTypes/Products/ATContentTypes/content/file.py
===================================================================
--- extras/Products.ATContentTypes/Products/ATContentTypes/content/file.py	(revision 54941)
+++ extras/Products.ATContentTypes/Products/ATContentTypes/content/file.py	(working copy)
@@ -105,4 +105,7 @@
     security       = ClassSecurityInfo()
 
+    def __init__(self, *args, **kwargs):
+        raise 'WeDidntWannaUseThisThingAnymoreException!'
+
     security.declareProtected(View, 'index_html')
     def index_html(self, REQUEST=None, RESPONSE=None):
