Index: extras/Plone/Products/CMFPlone/tests/PloneTestCase.py
===================================================================
--- extras/Plone/Products/CMFPlone/tests/PloneTestCase.py	(revision 28521)
+++ extras/Plone/Products/CMFPlone/tests/PloneTestCase.py	(working copy)
@@ -5,4 +5,5 @@
 # $Id$

+from plone.app.blob.tests import bbb
 from Products.PloneTestCase.ptc import *

@@ -24,4 +25,6 @@
     """

+    layer = bbb.plone
+
     def setRequestMethod(self, method):
         self.app.REQUEST.set('REQUEST_METHOD', method)
@@ -42,2 +45,4 @@
        something fancy on Your Branch (tm), put it here.
     """
+
+    layer = bbb.plone
