Index: extras/Plone/Products/CMFPlone/tests/PloneTestCase.py
===================================================================
--- extras/Plone/Products/CMFPlone/tests/PloneTestCase.py	(revision 17963)
+++ extras/Plone/Products/CMFPlone/tests/PloneTestCase.py	(working copy)
@@ -5,4 +5,5 @@
 # $Id$
 
+from plone.app.blob.tests import db   # needs to be imported first to set up ZODB
 from Products.PloneTestCase.ptc import *
 
@@ -13,5 +14,8 @@
 classImplements(PloneSite, ITestCasePloneSiteRoot)
 
-setupPloneSite(extension_profiles=['Products.CMFPlone:testfixture'])
+from plone.app.blob.tests.base import setupPackage
+setupPackage()
+
+setupPloneSite(extension_profiles=['Products.CMFPlone:testfixture', 'plone.app.blob:default'])
 
 from plone.protect.authenticator import AuthenticatorView
