Index: extras/plone.app.linkintegrity/plone/app/linkintegrity/tests/layer.py
===================================================================
--- extras/plone.app.linkintegrity/plone/app/linkintegrity/tests/layer.py	(revision 34037)
+++ extras/plone.app.linkintegrity/plone/app/linkintegrity/tests/layer.py	(working copy)
@@ -1,2 +1,3 @@
+from plone.app.blob.tests import bbb        # needs to be imported first
 from Testing.ZopeTestCase import installPackage
 from Products.Five import zcml, fiveconfigure
@@ -54,3 +55,3 @@


-integrity = PloneLinkintegrity(bases=[ptc_layer])
+integrity = PloneLinkintegrity(bases=[ptc_layer, bbb.plone])
